### DISCUSSION TITLE
公式与上文相隔过远的问题

### DISCUSSION THREAD

**Discussion by SiriusFang2020** [sikouhjw/gdutthesis#26] [Q&A] [answered]
你好，请问像图片里公式跟上文隔得特别远，但是跟下文就是正常的距离，这个问题该怎么解决呢
![4981646913766_ pic](https://user-images.githubusercontent.com/59632817/157658307-46e8072d-242f-43d8-ab1c-e8cfb9e774a7.jpg)
？

**Comment by sikouhjw** [comment, marked as answer]
公式前面是不是有空行？

**Comment by SiriusFang2020** [comment]
并没有呢
![image](https://user-images.githubusercontent.com/59632817/157661942-d9d092cc-5d26-4301-86eb-bee10c207d6f.png)

**Comment by sikouhjw** [comment]
两张截图并不相对应

**Comment by SiriusFang2020** [comment]
不好意思，截错图了，经测试确实是空行的问题，多谢🙏

**Comment by Yuly143982** [comment]
![1648451216(1)](https://user-images.githubusercontent.com/72240527/160344318-9199e2a8-b68c-4807-afb1-c0d090afd20b.png)
我的这个感觉间隔又太近了，还有表头与最后正文的间隔 距离看着有点远

**Reply by sikouhjw** [reply]
1. 公式与正文之间的间距，参考 https://github.com/sikouhjw/gdutthesis/discussions/34 ；
2. 浮动体与正文之间的间距，参考 https://github.com/sikouhjw/gdutthesis/discussions/50 。

**Reply by sikouhjw** [reply]
浮动体与正文的间距非常的复杂，LaTeX 的处理逻辑跟 Word 差太多。

表格与正文的间距，表头是正常的逻辑，而表尾是按照最后一行的基线去计算的，也就是不是按照最后一根线去计算的，我建议别纠结。

**Reply by Yuly143982** [reply]
嗯嗯，好的
